File: quasar/Quasar.DataTransfer/Usuarios/Responses/UsuarioResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Quasar.DataTransfer.Usuarios.Responses
{
    public class UsuarioResponse
    {
        public string Codigo { get; set; }
        public string Email { get; set; }
        public ClienteResponse Cliente { get; set; }
    }
}

File: quasar-front/src/app/shared/models/responses/usuario.response.ts
import { ClienteResponse } from './cliente.response';
export class UsuarioResponse {
  public codigo: string;
  public email: string;
  public cliente: ClienteResponse;

  constructor(params: Partial<UsuarioResponse>) {
    this.codigo = params.codigo || "";
    this.email = params.email || "";
    this.cliente = params.cliente || new ClienteResponse({});
  }
}
